#!/usr/bin/env python3
PACKAGE = "reconfigure"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("FR_hip_pos", double_t, 0, "FR hip pos", 0.00, -1.047, 0.872)
gen.add("FR_thigh_pos", double_t, 0, "FR thigh pos", 0.71184, -0.523, 3.926)
gen.add("FR_calf_pos", double_t, 0, "FR calf pos", -1.5705, -3.141, 3.141)

gen.add("FL_hip_pos", double_t, 0, "FL hip pos", 0.00, -1.047, 0.872)
gen.add("FL_thigh_pos", double_t, 0, "FL thigh pos", 0.71184, -0.523, 3.926)
gen.add("FL_calf_pos", double_t, 0, "FL calf pos", -1.5705, -3.141, 3.141)

gen.add("BR_hip_pos", double_t, 0, "BR hip pos", 0.00, -1.047, 0.872)
gen.add("BR_thigh_pos", double_t, 0, "BR thigh pos", 0.71184, -0.523, 3.926)
gen.add("BR_calf_pos", double_t, 0, "BR calf pos", -1.5705, -3.141, 3.141)

gen.add("BL_hip_pos", double_t, 0, "BL hip pos", 0.00, -1.047, 0.872)
gen.add("BL_thigh_pos", double_t, 0, "BL thigh pos", 0.71184, -0.523, 3.926)
gen.add("BL_calf_pos", double_t, 0, "BL calf pos", -1.5705, -3.141, 3.141)

exit(gen.generate(PACKAGE, "reconfigure", "position_reconfigure"))